jte/src/test/java/gg/jte/compiler/IoUtilsTest.java

```diff
@@ -0,0 +1,40 @@
+package gg.jte.compiler;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+class IoUtilsTest {
+    @Test
+    void shouldIdentifyTemplateFiles() {
+        Assertions.assertTrue(IoUtils.isTemplateFile("foo.jte"));
```

Comment:
Can you adjust the assertions to use AssertJ? That's what is currently used for all jte tests and it's a bit confusing if assertions are mixed throughout the project.

Reply:
Yup yup